SIL.WritingSystems/Resources/LanguageIndex.txt

```diff
@@ -15699,37 +15699,37 @@ gaw	PG	LA	Butelkud-Guntabak
 gaw	PG	LA	Garuh
 gaw	PG	LA	Nobanob
 gaw	PG	LA	Nobnob
-gax	ET	L	Oromo, Borana-Arsi-Guji
-gax	ET	LA	Afan Oromo
-gax	ET	LA	Southern Oromo
-gax	ET	LP	Galligna
-gax	ET	LP	Gallinya
-gax	KE	D	Arsi
-gax	KE	D	Boran
-gax	KE	D	Guji
-gax	KE	L	Borana
-gax	KE	LA	Booran
-gax	KE	LA	Boraan
-gax	KE	LA	Boran
-gax	KE	LA	Oromo
-gax	KE	LA	Southern Oromo
-gax	SO	D	Borana
-gax	SO	DA	Booran
-gax	SO	DA	Boran
-gax	SO	L	Oromo, Borana-Arsi-Guji
-gax	SO	LA	Southern Oromo
+gax	ET	!L	Oromo, Borana-Arsi-Guji
```

Comment:
No. There is no need/call for those offensive words that were removed from this file to exist in in our repository. This file is not the Ethnologue. It is a result of the query of the Ethnologue, for the purposes of a UI. The next time we build it, those lines and a few hundred others will be removed.